easybuild/easyconfigs/c/CGAL/CGAL-4.11-foss-2017b.eb

```diff
@@ -0,0 +1,40 @@
+name = 'CGAL'
+version = '4.11'
+
+homepage = 'http://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'strict': True}
+
+source_urls = ['https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-%(version)s']
+sources = [SOURCE_TAR_XZ]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Boost', '1.65.1'),
+    ('MPFR', '3.1.6'),
+    ('GMP', '6.1.2'),
+    ('libGLU', '9.0.0'),
+]
+
+builddependencies = [
+    ('CMake', '3.9.4'),
+    ('Eigen', '3.3.4'),
+    ('Qt5', '5.9.2'),
```

Comment:
Is it correct to include `Qt5` as a *build* dependency?

Also, since existing `CGAL` easyconfigs include `Qt` 4.x, maybe we should discriminate this easyconfig from the existing ones by mentioning `Qt5-5.9.2` in the `versionsuffix`?

Reply:
Qt5 is in build dependency because i could not compile openFOAM eb having qt5 from cgal and qt4 from Paraview (i managed to get the 2 modules load together, but gcc includes files from both qt ->function definition conflict) so i just put it in the build dep so cgal can compile his qt5 module but you can load it and use it to compile openfoam with qt4

Reply:
I'm not sure that's the right approach... Is CGAL going to find the correct `Qt` libraries?

Have you actually tested the OpenFOAM and ParaView installations with this in place?